safehttp/response.go

```diff
@@ -38,12 +38,13 @@ type Template interface {
 	Execute(wr io.Writer, data interface{}) error
 }
 
-// TemplateResponse bundles a Template with its data to be passed together to the
-// commit phase. This will be passed when the commit phase is initiated from
-// ResponseWriter.WriteTemplate.
+// TemplateResponse bundles a Template with its data and names to function
+// mappings to be passed together to the commit phase. A TemplateResponse will
+// be created when the commit phase is initiated from ResponseWriter.WriteTemplate.
```

Comment:
```suggestion
// TemplateResponse bundles a Template with its data and names to function
// mappings to be passed together to the commit phase.
```

Reply:
I left out information that doesn't seem to benefit the user (what are they supposed to do about it?).

Reply:
Right, that's implementation details.